src/item.cpp

```diff
@@ -1556,6 +1546,16 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             info.emplace_back( "DESCRIPTION", _( "* This item is <bad>not repairable</bad>." ) );
         }
 
+        if( !conductive () ) {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item <good>does not conduct</good> electricity." ) ) ) );
+            }
+            else if( has_flag( "CONDUCTIVE" ) ) {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item effectively <bad>conducts</bad> electricity, as it has no guard." ) ) ) );
+            }
+            else {
+                info.push_back( iteminfo( "BASE", string_format( _( "* This item <bad>conducts</bad> electricity." ) ) ) );
+            }
+
```

Comment:
Everything has an extra indent, the closing bracket should be directly below the if, right?